godtools/Views/BaseTractView.swift

```diff
@@ -14,19 +14,34 @@ class BaseTractView: UIView {
     
     var data: XMLIndexer?
     var colors: TractColors?
-    var configurations = TractConfigurations()
+    var configurations: TractConfigurations?
```

Comment:
Actually to take this one step further. The first three (data, colors, configurations) are not used anywhere outside this class, so you don't need to store them as properties on the class at all unless I'm missing something?